packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -151,6 +159,14 @@ export async function getConsolidated({
       dateTo,
     });
     const hasResources = bundle.entry && bundle.entry.length > 0;
+
+    if (conversionType === "json") {
+      return uploadConsolidatedJsonAndReturnUrl({
+        patient,
+        bundle,
+        filters,
+      });
+    }
```

Comment:
This should prob be the second last command in this function, no?

Like, the way it is now, we're not sending analytics, for example.

I'd update `shouldCreateMedicalRecord` so its `false` if `conversionType` is `json` and have this as the last command before the return.

Reply:
good catch